engine/apps/api/views/alert_group.py

```diff
@@ -290,8 +296,6 @@ class AlertGroupView(
         "filters": [RBACPermission.Permissions.ALERT_GROUPS_READ],
         "silence_options": [RBACPermission.Permissions.ALERT_GROUPS_READ],
         "bulk_action_options": [RBACPermission.Permissions.ALERT_GROUPS_READ],
-        "create": [RBACPermission.Permissions.ALERT_GROUPS_WRITE],
-        "update": [RBACPermission.Permissions.ALERT_GROUPS_WRITE],
```

Comment:
we don't have a `create` or `update` method defined on `AlertGroupView`, no need for these permissions here